civihr_employee_portal/src/Hook/ModuleImplementsAlter/ImplementationAlterer.php

```diff
@@ -20,6 +20,22 @@ public function shouldAlter($hookName);
    *
    * @return void
    */
-  public function alter(array &$implementations);
+  abstract public function alter(array &$implementations);
 
+  /**
+   * @param array $implementations
+   * @param string $moduleName
+   */
+  protected function moveToEnd(array &$implementations, $moduleName) {
+    if (!isset($implementations[$moduleName])) {
+      $serr = sprintf('Cannot re-order as module "%s" is not set', $moduleName);
```

Comment:
Can you rename this variable to something more readable. Maybe `error` or something better